OpenRA.Mods.Common/Warheads/LeaveSmudgeWarhead.cs

```diff
@@ -18,6 +18,7 @@
 
 namespace OpenRA.Mods.Common.Warheads
 {
+	[Desc("Spawn a permanent image.")]
```

Comment:
It's not really spawning an image but interacting with the `SmudgeLayer` by adding a smudge to it.

Reply:
I'm not really familiar with SmudgeLayer. Should the description be "Interacts with `SmudgeLayer`."

Reply:
Maybe just something like "Creates terrain smudges on impact."

Reply:
I'm going with "Creates a smudge in `SmudgeLayer`."